src/main/resources/db/migration/V15__add_offence_table.sql

```diff
@@ -0,0 +1,17 @@
+drop table if exists offence;
+
+create table offence
+(
+    id                          serial primary key,
+    draft_adjudication_fk_id    bigint references draft_adjudications (id),
+    offence_code                integer       not null,
```

Comment:
Out of interest do we have list of offence codes - just wondering if there should be a reference table for them, or if we are just leaving it free form

Reply:
I was thinking it's free form in the API - the UI will contain that lookup. It means we don't have to update the API if/when offence codes are added.

My initial thinking was that the API should contain this information, but it's the maintenance that's the problem. If you have an enum then the UI has to have a version of that enum, and map that to the UI values.